rllib/policy/sample_batch.py

```diff
@@ -49,6 +49,10 @@ class SampleBatch(dict):
     OBS_EMBEDS = "obs_embeds"
     T = "t"
 
+    # decision transformer
+    RETURNS_TO_GO = "returns_to_go"
+    ATTENTION_MASKS = "attention_masks"
+
```

Comment:
Arguments could be made for putting it here, or putting it elsewhere like in `dt.py` which doesn't exist yet.

Reply:
we're putting this here.